src/irmin/tree_intf.ml

```diff
@@ -363,7 +369,10 @@ module type S = sig
   val counters : unit -> counters
   val dump_counters : unit Fmt.t
   val reset_counters : unit -> unit
-  val inspect : t -> [ `Contents | `Node of [ `Map | `Key | `Value | `Pruned ] ]
+
+  val inspect :
+    t ->
+    [ `Contents | `Node of [ `Map | `Key | `Value | `Portable_dirty | `Pruned ] ]
```

Comment:
Probably worth documenting these at some point, especially if they're going to appear in exception messages when interacting with trees.

Reply:
👍 

Opened an issue here https://github.com/mirage/irmin/issues/1759